scripts/babel-6/rewrite-modules.js

```diff
@@ -76,26 +93,26 @@ module.exports = function(babel) {
    */
   function transformJestHelper(path, state) {
     var calleePath = path.get('callee');
-    var args = getArguments(path);
+    var args = path.get('arguments').map((p) => p.node);
+    if (!args.length) {
+      return;
+    }
+    var moduleArg = args[0];
     if (
-      args &&
-      args.type === 'StringLiteral' &&
+      moduleArg &&
+      moduleArg.type === 'StringLiteral' &&
       calleePath.node &&
-      (
-        t.isIdentifier(calleePath.node.property, {name: 'dontMock'}) ||
-        t.isIdentifier(calleePath.node.property, {name: 'mock'}) ||
-        t.isIdentifier(calleePath.node.property, {name: 'genMockFromModule'})
-      )
+      isJestProperty(t, calleePath.node.property)
     ) {
-      var module = mapModule(state, args.value);
+      var module = mapModule(state, moduleArg.value);
       if (module) {
-        args = t.stringLiteral(module);
+        args[0] = t.stringLiteral(module);
       }
     }
     path.replaceWith(
```

Comment:
*but* `path.get('arguments')[0].replaceWith(t.stringLiteral(module))` does work (since that's a `path`). Might rework a few other lines to account for that…